### PR TITLE
Port to Unified XM 4.5

### PR DESCRIPTION
**Pull request by chamons**

### CODE REVIEW

File: AppDelegate.cs [hunk 1]
```diff
@@ -289,6 +290,9 @@ void LaunchDocumentationUpdate (Product [] products, string informative)
 			mergeController.ShowWindow (this);
 			mergeController.Window.Center ();
 		}
+
+		[DllImport ("/usr/lib/libobjc.dylib", EntryPoint = "objc_msgSend")]
```
Comment (abock) [inline AppDelegate.cs]: `[DllImport (Constants.ObjectiveCLibrary, EntryPoint = "objc_msgSend")]`

**Review by abock — CHANGES_REQUESTED** [review]
Wooo

File: AppDelegate.cs (hunk 1 above)
Comment (chamons) [reply AppDelegate.cs]: I didn't know that we had constants publicly accessible....

Comment (abock) [reply AppDelegate.cs]: Yeah, they are actually super useful 😺

**Review by abock — APPROVED** [review]

**Pull request merged**